index.d.ts

```diff
@@ -0,0 +1,259 @@
+// Type definitions for nouislider v10.1.0
```

Comment:
Could you remove these comments? You can add the author information to `readme.md`.